.env.template

```diff
@@ -1,4 +1,4 @@
-# get the complete env from other volunteers, please
+AIRFLOW_HOME=/opt/airflow
 BIGQUERY_PROJECT=pycontw-225217
```

Comment:
nitpick: I feel this might be something we should make it configurable.  🤔

Reply:
Is AIRFLOW_HOME redundant?

Reply:
no, I mean BIGQUERY_PROJECT

Reply:
developer may use their own GCP project for testing

Reply:
Yep, that's why I think making it configurable might be a good idea. WDYT?